### ISSUE TITLE
double click doesn't work in devtools protocol

### ISSUE THREAD

**Issue by mgrybyk** [webdriverio/webdriverio#5904]
**Environment (please complete the following information):**
 - **WebdriverIO version:** 6.5.0
 - **Automation Protocol** devtools

**Config of WebdriverIO**
`automationProtocol: "devtools"`

**Describe the bug**
double click doesn't work as expected with devtools implementation of `performActions`

**To Reproduce**
```
browser.url("http://demo.guru99.com/test/simple_context_menu.html");
let elem = browser.$("//button[text()='Double-Click Me To See Alert']");
elem.waitForClickable({ timeout: 2000 });
elem.doubleClick();
browser.pause(5000);
```

alert should appear in case of successful doubleclick

**Expected behavior**
double click should be treated as double click not just to separate clicks

**Additional context**
`performActions` works as expected, it clicks two times in a row on the element. I think it's an issue with how Puppeteer sends click events with synthetic MouseEvent.

**Comment by mgrybyk** [comment]
I have a suggestion on how to fix doubleclick.

1. adopt elementClick to accept options
2. create doubleClick command that would trigger elementClick and pass `{ clickCount : 2 }` 
3. call devtools doubleClick command in webdriverio doubleClick command if automation protocol is devtools

I can do that if the approach is acceptable
@christian-bromann

**Comment by christian-bromann** [comment]
Sounds like a plan, thanks @mgrybyk !

**Comment by christian-bromann** [comment]
fixed in #6927
